projects/Mallard/src/screens/article-screen.tsx

```diff
@@ -92,11 +96,17 @@ export const ArticleScreen = ({
                     >
                         <Article
                             article={elements}
-                            kicker={'Kicker 🥾'}
-                            headline={title}
-                            byline={'Byliney McPerson'}
-                            standfirst={`Is this delicious smoky dip the ultimate aubergine recipe – and which side of the great tahini divide are you on?`}
-                            image={imageURL}
+                            kicker={(frontArticle && frontArticle.kicker) || ''}
+                            headline={
+                                (frontArticle && frontArticle.headline) || title
+                            }
+                            byline={
+                                (frontArticle && frontArticle.byline) || byline
```

Comment:
```suggestion
                                byline
```

let's not use stale data if we have the latest one

Reply:
it's not stale, if it's in the front response it has been deliberately modified.